src/App.tsx

```diff
@@ -1,172 +1,30 @@
-import { Loader } from './components/Loader';
-
 import './App.scss';
+import {
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { Navigation } from './components/Navigation/Navigation';
+import { PeoplePage } from './pages/PeoplePage';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
 
 export const App = () => (
   <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
+    <Navigation />
 
     <main className="section">
       <div className="container">
```

Comment:
Is your router inside a section?

Let's move inside a router.
Create a `Layout` component and wrap all your routes in the `Layout` route.

```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside.